package.json

```diff
@@ -12,7 +12,7 @@
 	},
 	"scripts": {
 		"wizard": "node ./src/index.js",
-		"start": "node ./src/index.js && node ./src/bot/index.js",
+		"start": "cp -r buffer-layout/* node_modules/@solana/buffer-layout/ node ./src/index.js && node ./src/bot/index.js",
```

Comment:
This needs && between cp and node.

`"start": "cp -r buffer-layout/* node_modules/@solana/buffer-layout/ && node ./src/index.js && node ./src/bot/index.js",

Reply:
good call, I eyeballed it <3

Reply:
@arbcaptain can you approve, I cannot